examples/ink/index.ts

```diff
@@ -60,7 +125,7 @@ const psp22 = getInkClient(contracts.psp22)
 
   if (response.result.success) {
     console.log(increaseAllowance.decode(response.result.value))
-    console.log(psp22.event.filter(ADDRESS.psp22, response.events))
+    // console.log(psp22.event.filter(ADDRESS.psp22, response.events))
```

Comment:
should it be removed?

Reply:
it's temporary until events v5 is fixed (coming up just now :D)